vertx-pg-client/README.adoc

```diff
@@ -170,15 +170,15 @@ The *Reactive Postgres Client* currently supports the following data types
 |&#10004;
 
 |`JSON`
-|`i.r.p.data.Json`
+|`Json`
```

Comment:
now should be `Object`

Reply:
updated with https://github.com/eclipse-vertx/vertx-sql-client/pull/324/commits/a09288db275dc4b19adf36b84b184a31e56f223e.